walking:
  engine:
    double_support_ratio: 0.25
    first_step_swing_factor: 1
    foot_distance: 0.0892010830100436
    foot_rise: 0.07
    freq: 0.5
    trunk_height: 0.183555759791475
    trunk_phase: -0.0012469948672716
    trunk_pitch: 0.1
    trunk_swing: 1.4
    trunk_x_offset: 0.0233908598440774
    trunk_y_offset: -0.000632849143066137
    trunk_z_movement: 0.0
    early_termination_at: [0.0, 7.0, 0.0]
    first_step_trunk_phase: 0
    foot_apex_phase: 0.5
    foot_overshoot_phase: 1
    foot_overshoot_ratio: 0.0
    foot_put_down_phase: 1
    foot_z_pause: 0
    trunk_pause: 0
    trunk_pitch_p_coef_forward: 0
    trunk_pitch_p_coef_turn: 0

    kick_length: 0.09
    kick_vel: 0.2
    kick_phase: 0.75

    foot_put_down_z_offset: 0.0

  node:
    # update frequency of the engine
    engine_freq: 125

    # parameters for bioIK
    ik_timeout: 0.01

    debug_active: True

    trunkYOnlyInDoubleSupport: False

    max_step_x: 1000
    max_step_y: 1000
    max_step_xy: 1000
    max_step_z: 1000
    max_step_angular: 1000

    vel: -1
    acc: -1
    pwm: -1

    imu_active: False
    imu_pitch_threshold: 0.19
    imu_roll_threshold: 0.4
    imu_pitch_vel_threshold: 1.3
    imu_roll_vel_threshold: 5.7
    pause_duration: 3.0

    phase_reset_active: False
    ground_min_pressure: 1.5
    phase_reset_phase: 0.25
    joint_min_effort: 3
    effort_phase_reset_active: False

    x_speed_multiplier: 1
    y_speed_multiplier: 1
    yaw_speed_multiplier: 1

  pid_trunk_fused_pitch:
    p: 0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: False

  pid_trunk_fused_roll:
    p: 0
    i: 0
    d: 0
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: False
    publish_state: False
